src/pages-and-resources/discussions/DiscussionsSettings.test.jsx

```diff
@@ -127,9 +129,9 @@ describe('DiscussionsSettings', () => {
       // content has been loaded - prior to proceeding with our expectations.
       await waitForElementToBeRemoved(screen.queryByRole('status'));
 
-      await waitFor(() => {
-        userEvent.click(queryByLabelText(container, 'Select Piazza'));
-        userEvent.click(queryByText(container, messages.nextButton.defaultMessage));
+      await waitFor(async () => {
```

Comment:
For me it passes without this `waitFor` - I think we can remove it. Generally interactions like `user.click` should not be inside a `waitFor` - only the result like `expect(...).toBe...` should be.